spotbugs/etc/messages.xml

```diff
@@ -9033,6 +9041,129 @@ Using floating-point variables should not be used as loop counters, as they are
         ]]>
     </Details>
   </BugPattern>
+  <BugPattern type="PI_DO_NOT_REUSE_PUBLIC_IDENTIFIERS_CLASS_NAMES">
+        <ShortDescription>Do not reuse public identifiers from JSL as class name</ShortDescription>
```

Comment:
"name" -> "names"

Reply:
This is from another already accepted PR https://github.com/spotbugs/spotbugs/pull/2511, which got here when merging master to this branch. I proposed some changes to the messages in https://github.com/spotbugs/spotbugs/pull/2689.

Reply:
I am skipping it.